jobs/webcompat-kb/webcompat_kb/main.py

```diff
@@ -857,7 +869,7 @@ def fetch_update_history(self, all_bugs):
         ) and self.history_fetch_completed:
             filtered_existing = self.filter_relevant_history(existing_bugs_history)
             filtered_records = filtered_existing + filtered_new_history
-            return self.update_history(filtered_records)
+            # return self.update_history(filtered_records)
```

Comment:
Need to re-enable all this commented out code :)

Reply:
ah yes, thanks! 😄